SofaKernel/framework/sofa/helper/system/PluginManager.h

```diff
@@ -168,7 +168,7 @@ class SOFA_HELPER_API PluginManager
     void init();
     void init(const std::string& pluginPath);
 
-    std::string findPlugin(const std::string& pluginName, const std::string& suffix = getDefaultSuffix(), bool ignoreCase = true);
+    std::string findPlugin(const std::string& pluginName, const std::string& suffix = getDefaultSuffix(), bool ignoreCase = true, int maxRecursiveDepth = 4);
```

Comment:
magic number or is 4 really meaning something?

Reply:
Changed it to 6 :wink: